backend/src/layer/trigger/scheduling/scheduler.rs

```diff
@@ -177,14 +177,14 @@ mod tests {
     use crate::layer::trigger::scheduling::mocks::{ImageReviewMock, MensaParseMock};
 
     use super::*;
-    use tracing::Level;
+    // use tracing::Level;
     #[tokio::test]
     async fn test_scheduling() {
-        let subscriber = tracing_subscriber::FmtSubscriber::builder()
-            .with_max_level(Level::TRACE)
-            .finish();
-        tracing::subscriber::set_global_default(subscriber)
-            .expect("Setting default subscriber failed");
+        // let subscriber = tracing_subscriber::FmtSubscriber::builder()
```

Comment:
Why did you comment this?

Reply:
these were the spam comments you complaint about

Reply:
Spam comments?

Reply:
*log messegas

Reply:
i can remove these completely

Reply:
Yes please